tools/introspection/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+    <name>iceoryx_introspection</name>
+    <version>2.0.3</version>
+    <description>Eclipse iceoryx inter-process-communication (IPC) middleware introspection client</description>
+    <maintainer email="iceoryx-oss-support@apex.ai">Eclipse Foundation, Inc.</maintainer>
+    <license>Apache 2.0</license>
+    <url type="website">https://iceoryx.io</url>
+    <url type="bugtracker">https://github.com/eclipse-iceoryx/iceoryx/issues</url>
+    <url type="repository">https://github.com/eclipse-iceoryx/iceoryx</url>
+
+    <buildtool_depend>cmake</buildtool_depend>
+    <buildtool_depend>git</buildtool_depend>
```

Comment:
We don't need git as build tool dependency here since there are no external projects included.

```suggestion
```